### PR TITLE
[aice/v1.21.0]Fix running RedHatAI/Qwen2-7B-Instruct-FP8

### PR DESCRIPTION
**Pull request by ccrhx4**
… of range (expected to be in range of [-1, 0], but got 1) (#1334)

This is a bug fix for https://jira.habana-labs.com/browse/SW-229993

Validated the same model and now it runs OK
2025-05-29:01:07:32,806 INFO [lm_eval.loggers.evaluation_tracker:290] Saving per-sample results for: gsm8k
vllm
(pretrained=/mnt/weka/data/pytorch/llama3/Meta-Llama-3-8B-Instruct-AutoFP8/,tensor_parallel_size=1,distributed_executor_backend=mp,trust_remote_code=true,max_model_len=4096,use_v2_block_manager=True,dtype=bfloat16,max_num_seqs=128), gen_kwargs: (None), limit: 256.0, num_fewshot: 5, batch_size: auto |Tasks|Version| Filter |n-shot| Metric | |Value | |Stderr|

|-----|------:|----------------|-----:|-----------|---|-----:|---|-----:| |gsm8k| 3|flexible-extract| 5|exact_match|↑ |0.7383|± |0.0275| | | |strict-match | 5|exact_match|↑ |0.7422|± |0.0274|

FILL IN THE PR DESCRIPTION HERE

FIX #xxxx (*link existing issues this PR will resolve*)

**BEFORE SUBMITTING, PLEASE READ <https://docs.vllm.ai/en/latest/contributing/overview.html>** (anything written below this line will be removed by GitHub Actions)

### CODE REVIEW

**Review by czhu15 — APPROVED** [review]
LGTM
